common/src/main/java/io/homeassistant/companion/android/common/data/integration/impl/IntegrationRepositoryImpl.kt

```diff
@@ -380,6 +380,31 @@ class IntegrationRepositoryImpl @Inject constructor(
         }.toTypedArray()
     }
 
+    override suspend fun getEntity(entityId: String): Entity<Any>? {
```

Comment:
Probably want to throw an exception if we don't have the entity.
```suggestion
    override suspend fun getEntity(entityId: String): Entity<Any> {
```